app/src/main/java/org/simple/clinic/drugs/selectionv2/PrescribedDrugEvents.kt

```diff
@@ -7,3 +7,5 @@ import java.util.UUID
 data class ProtocolDrugSelected(val drugName: String, val prescribedDrug: PrescribedDrug?) : UiEvent
```

Comment:
Can we rename this event to `ProtocolDrugClicked`? Because the user doesn't "select" anything yet. It just sounds more correct to me.